packages/@sanity/form-builder/src/inputs/ObjectInput/fieldGroups/Group.tsx

```diff
@@ -0,0 +1,88 @@
+import React, {forwardRef, useEffect} from 'react'
```

Comment:
This file should probably be renamed to GroupTab.tsx so that it matches the component name